flight/tasks/sun.py

```diff
@@ -2,21 +2,34 @@
 
 import time
 
-from apps.sun import compute_body_sun_vector, in_eclipse
+from apps.sun import compute_body_sun_vector_from_lux, in_eclipse, read_light_sensors
 from core import TemplateTask
 from core import state_manager as SM
 from core.data_handler import DataHandler as DH
-from hal.configuration import SATELLITE
 
 
 class Task(TemplateTask):
 
     name = "SUN"
     ID = 0x11
 
-    data_keys = ["time", "x", "y", "z", "eclipse"]
+    data_keys = [
+        "time",
+        "status",
+        "x",
+        "y",
+        "z",
+        "eclipse",
+        "xp",
+        "xm",
+        "yp",
+        "ym",
+        "zp",
+    ]
 
-    # Fake starting sun vector
+    THRESHOLD_ILLUMINATION_LUX = 3000
+
+    status = False
     sun_vector = [1, 0, 0]
```

Comment:
why start with non-zero vector here?

Reply:
Good catch. Missed legacy code. Changed the initial status as well.